### DISCUSSION TITLE
Alternative resource for "country picker" example in docs

### DISCUSSION THREAD

**Discussion by creativesuspects** [leekelleher/umbraco-contentment#231] [Idea]
Hi @leekelleher! Maybe you could reference the following resource on the [data list docs](https://github.com/leekelleher/umbraco-contentment/blob/develop/docs/editors/data-list.md) in the **Country picker** example. Especially since the existing [XML resource](https://madskristensen.net/posts/files/countries.xml) uses an invalid SSL certificate.

https://github.com/stefangabos/world_countries/

> Constantly updated lists of world countries, territories and areas of geographical interest, with associated alpha-2, alpha-3 and numeric codes as defined by the ISO 3166-1 standard, published and maintained by the [International Organization for Standardization](https://www.iso.org/iso-3166-country-codes.html), available in CSV, JSON, PHP, SQL and XML formats, in multiple languages and with national flags included. Also available are the ISO 3166-2 codes used for identifying the principal subdivisions (e.g., provinces or states) of all countries coded in ISO 3166-1.

**Comment by leekelleher** [comment]
Hi @creativesuspects, thank you for the suggestion. 👍 

> Especially since the existing [XML resource](https://madskristensen.net/posts/files/countries.xml) uses an invalid SSL certificate.

I've just checked [Mads Kristensen's website](https://www.madskristensen.net/), and it turns out that the non-www domain has the expired SSL certificate, but the www domain has a valid one. So <https://www.madskristensen.net/posts/files/countries.xml> should be valid now. I'll update the docs.


> <https://github.com/stefangabos/world_countries/>

Thanks for letting me know about this, it looks to be a very comprehensive resource!

The main thing for the XML data-source example would be that the URL is publicly accessible, but the [World countries webpage](http://stefangabos.github.io/world_countries/) seems targeted to downloading the dataset. Although it could be hot linked from the GitHub repo, e.g. <https://raw.githubusercontent.com/stefangabos/world_countries/master/data/countries/en/countries.xml>

The combined languages dataset looks interesting, <https://github.com/stefangabos/world_countries/blob/master/data/countries/_combined/countries.xml> - gets me thinking about multi-lingual support for Data List. 🤔 😁

**Reply by creativesuspects** [reply]
Multi-lingual support for something like a country data list could be very useful! A single data-source that includes all the language variants may not always be easy to find. Maybe instead allow a _magic string_ in the URL to include the culture name (en-US) or language ISO (en)?

{CultureName}
{TwoLetterISOLanguageName}
{ThreeLetterISOLanguageName}
